app/helpers/announcements_helper.rb

```diff
@@ -4,10 +4,11 @@ def announcement_hidden?(announcement)
   end
 
   def current_announcement
+    return @current_announcement if defined? @current_announcement
     @current_announcement = Announcement.current
   end
 
   def no_announcement
-  	current_announcement.nil? or announcement_hidden?(current_announcement)
+    current_announcement.none or announcement_hidden?(current_announcement)
```

Comment:
I don't think `none` what you want here. All boolean methods in Rails terminate with a question mark.
Check the docs for what this is actually doing.http://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html